content/tagging/assigning_tags.md

```diff
@@ -13,25 +13,195 @@ further_reading:
   text: Learn how to use tags in Datadog
 ---
 
-Tagging is used throughout the Datadog product to make it easier to subset and query the machines and metrics that you have to monitor. Without the ability to assign and filter based on tags, finding the problems that exist in your environment and narrowing them down enough to discover the true causes would be extremely difficult. [Discover Datadog's tagging best practices before going further][61].
+Tagging is used throughout Datadog to query the machines and metrics you monitor. Without the ability to assign and filter based on tags, finding problems in your environment and narrowing them down enough to discover the true causes would be extremely difficult. Discover [how to use][61] tagging in Datadog before going further.
 
-## How to assign tags
-There are four primary ways to assign tags: inherited from the [integration][1], in the configuration, in the UI, and using the API, though the UI and API only allow you to assign tags at the host level. The recommended method is to rely on the [integrations][1] or via the configuration files.
+## Where to assign tags
+There are several places you can assign tags: using configuration files, [environment variables][80], the UI, [API][65], [DogStatsD][75], and inheriting from the [integrations][1]. It is recommended to rely on the configuration files and integration inheritance for most of your tagging needs.
 
-### Inheriting tags from an integration
+### Configuration Files
 
-The easiest method for assigning tags is to rely on the integration. Tags assigned to your Amazon Web Services instances, Chef recipes, and more are all automatically assigned to the hosts and metrics when they are brought in to Datadog.
+You can configure the host tags submitted by the Agent inside `datadog.yaml`. The tags for the [integrations][1] installed with the Agent are configured via yaml files located in the **conf.d** directory of the Agent install. To locate the configuration files, refer to [this page][59].
 
-The following [integrations][1] sources create tags automatically in Datadog:
+In YAML files, use a tag dictionary with a list of tags you want assigned at that level. Any tag you assign to the Agent is applied to every integration on that Agent's host.
```

Comment:
r/`is applied to every integration on that Agent's host.` / `is applied to every integration on that Agent's host and so to all their corresponding metrics, traces, and logs.`